content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,142 @@
+
+### Enum value
+
+For variables which accept a known value out of a set, i.e., an enum value, SDK implementations MAY support additional values not listed here.
+For variables accepting an enum value, if the user provides a value the SDK does not recognize, the SDK MUST generate a warning and gracefully ignore the setting.
+
+If a null object (empty, no-op) value is acceptable, then the enum value representing it MUST be `"none"`.
+
+### Duration
+
+Any value that represents a duration, for example a timeout, MUST be an integer representing a number of
+milliseconds. The value is non-negative - if a negative value is provided, the SDK MUST generate a warning,
+gracefully ignore the setting and use the default value if it is defined.
+
+For example, the value `12000` indicates 12000 milliseconds, i.e., 12 seconds.
+
+
+
+## Jaeger Exporter
+
+**Status**: [Stable](document-status.md)
+
+The `OTEL_EXPORTER_JAEGER_PROTOCOL` environment variable
+MAY by used to specify the transport protocol.
+The value MUST be one of:
+
+- `http/thrift.binary` - [Thrift over HTTP][jaeger_http]
+- `grpc` - [gRPC][jaeger_grpc]
+- `udp/thrift.compact` - [Thrift with compact encoding over UDP][jaeger_udp]
+- `udp/thrift.binary` - [Thrift with binary encoding over UDP][jaeger_udp]
+
+[jaeger_http]: https://www.jaegertracing.io/docs/latest/apis/#thrift-over-http-stable
+[jaeger_grpc]: https://www.jaegertracing.io/docs/latest/apis/#protobuf-via-grpc-stable
+[jaeger_udp]: https://www.jaegertracing.io/docs/latest/apis/#thrift-over-udp-stable
+
+The default transport protocol SHOULD be `http/thrift.binary` unless
+SDKs have good reasons to choose other as the default
+(e.g. for backward compatibility reasons).
+
+Environment variables specific for the `http/thrift.binary` transport protocol:
+
+
+- OTEL_EXPORTER_JAEGER_ENDPOINT 
+ 
+ Full URL of the [Jaeger HTTP endpoint][jaeger_collector] 
+example: `http://localhost:14268/api/traces`
+- OTEL_EXPORTER_JAEGER_TIMEOUT 
+  Maximum time (in milliseconds) the Jaeger exporter will wait for each batch export 
+ example: 10000
+- OTEL_EXPORTER_JAEGER_USER
+ Username to be used for HTTP basic authentication 
+- OTEL_EXPORTER_JAEGER_PASSWORD 
+ Password to be used for HTTP basic authentication
+ 
+ 
+Environment variables specific for the `grpc` transport protocol:
+
+
+- OTEL_EXPORTER_JAEGER_ENDPOINT 
+ URL of the [Jaeger gRPC endpoint][jaeger_collector]
+ example: `http://localhost:14250` 
+- OTEL_EXPORTER_JAEGER_TIMEOUT  
+ Maximum time (in milliseconds) the Jaeger exporter will wait for each batch export 
+ example: 10000
+- OTEL_EXPORTER_JAEGER_USER    
+ Username to be used for HTTP basic authentication 
+- OTEL_EXPORTER_JAEGER_PASSWORD | Password to be used for HTTP basic authentication 
+
+
+Environment variables specific for the `udp/thrift.compact` transport protocol:
+
+-  OTEL_EXPORTER_JAEGER_AGENT_HOST  
```

Comment:
Variables are expressed as
## 'OTEL_EXPORTER_JAEGER_AGENT_HOST'
not
- 'OTEL_EXPORTER_JAEGER_AGENT_HOST'